### ISSUE TITLE
[A11Y] Visible hidden text in probe screen

### ISSUE THREAD

**Issue by bwbroersma** [bikkelhart/internet-nl#61]
`probes auto-redirect` ("Loading score for:") was a hidden text (only for screenreaders) before, that is now visible.

In main it is hidden:
https://github.com/bikkelhart/internet-nl/blob/d3290df0e49f4f48a0a6762255fa3f163eb3a755/interface/templates/base-probe.html#L34-L36

In redesign it is always visible:
https://github.com/bikkelhart/internet-nl/blob/eb9584ec7803339b60726e3a8a0d0ef7d9a0f3d6/interface/templates/base-probe.html#L36-L40

![Image](https://github.com/user-attachments/assets/ab69b51e-fa63-4935-b262-588a1c7b3b0a)

**Comment by Khdulkadir** [comment]
@bwbroersma probes auto-redirect isnt equal to ("Loading score for:"). Which one of these needs to be only visible for screenreaders?

        <span class="h5 color-secondary">{% include "string.html" with name="base_probe_score_loading_newstring" %}</span>


        {% include "string.html" with name="probes auto-redirect" %}

**Comment by bwbroersma** [comment]
Okay, so I should have looked up the strings:
`base_probe_score_loading_newstring = "Loading score for:"`
`probes auto-redirect = "You will be automatically redirected to the results page when all tests are finished."`

https://github.com/bikkelhart/internet-nl/blob/7cb9a8d5a4ff27b4038be87119fe32d8697339c7/interface/templates/base-probe.html#L30

Which is both a new string (it's in the name), and not used before.
Maybe the text is a bit weird, since there are now two ":" in the text.
But no A11Y issue.

I am unsure about this change:
https://github.com/bikkelhart/internet-nl/blob/7cb9a8d5a4ff27b4038be87119fe32d8697339c7/interface/templates/base-probe.html#L46
>  aria-live region must be visible if you want its content changes to be announced

source: https://stackoverflow.com/a/50748478

Seems logical.

Also, there used to be an `h1` on the page:
https://github.com/bikkelhart/internet-nl/blob/d3290df0e49f4f48a0a6762255fa3f163eb3a755/interface/templates/base-probe.html#L28-L31

Which changed to a span:
https://github.com/bikkelhart/internet-nl/blob/30de38964b4f67228ce84de1c294b9691bda8a58/interface/templates/base-probe.html#L31-L34

Are there valid reasons to make this semantic change?

Maybe also add an [`hgroup`](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/hgroup) to the leading `<span class="h5 color-secondary">` if it is part of the heading?

---
Like discussed:
- h1 is in outer level
- hgroup is already there

**Comment by Khdulkadir** [comment]
I just did some checking on:

```
<li class="probing icon h7 color-secondary">
        Bereikbaar via modern adres?
        <span id="ipv6-summary" class="visually-hidden" aria-live="polite">
            Bezig...
         </span>
  </li>
```

And apparently the current code is correct, because the span is only visually hidden not actually hidden, it also said the same in the stackoverflow comment you linked: 

> The short answer is no. an aria-live region must be visible if you want its content changes to be announced.
> 
> You may read [this question](https://stackoverflow.com/questions/50570426/toggle-aria-live-when-announcement-finishes) where I give a small trick: show the element a few seconds, long enough to let the screen reader read the message, and then hide again. However you must show the message at least for 3-5 seconds because some screen readers cut of before the end if you hide the element while it is still being spoken.
> 
> IF showing the message for that long is unacceptable, you can still put it off-screen, by using a little CSS like below. Note that many frameworks already have classes like .visually-hidden, .sr-only, etc. with a similar code. If you are using one of them, use what they define.
> 
> .visually-hidden {
> top:0;
> left:-2px;
> width:1px;
> height:1px;
> position:absolute;
> overflow:hidden;
> }
